stdlib/public/core/StringLegacy.swift

```diff
@@ -238,7 +231,7 @@ extension String {
 
   // FIXME: can't just use a default arg for radix below; instead we
   // need these single-arg overloads <rdar://problem/17775455>
-  
+
```

Comment:
btw, if you add `?w=0` to any url that provides a diff it will supress any whitespace changes. It will even suppress changes that are "only" re-indenting (which is actually pretty relevant to this patch, which is making a lot of if'd code unconditional).

e.g. 

https://github.com/apple/swift/pull/5077/commits/35c19fc77b7a153a598667e0ce965160f840a5c8?w=0

(checkout StringComparable for a good example)